NEWS.md

```diff
@@ -91,12 +101,12 @@
 * `scale_continuous()` uses a more correct check for numeric values.
 
 * NaN is correctly recognised as a missing value by the gradient palettes
-  (ggplot2#1482).
+  ([ggplot2-1482](https://www.github.com/tidyverse/ggplot2/issues/1482)).
   
 # scales 0.3.0
 
 * `rescale()` preserves missing values in input when the range of `x` is
-  (effectively) 0 (ggplot2#985).
+  (effectively) 0 ([ggplot2-985](https://www.github.com/tidyverse/ggplot2/issues/985)).
 
 * Continuous colour palettes now use `colour_ramp()` instead of `colorRamp()`.
   This only supports interpolation in Lab colour space, but is hundreds of
```

Comment:
Is there a word missing here? "is hundreds of…" If no (which is totally possible), toss in a period?

Reply:
not sure why it's not showing up here but it does continue to say "hundreds of times faster." i've double checked the file.

Reply:
I believe you! The internet is weird.